src/api/setting.py

```diff
@@ -16,7 +16,7 @@
 USE_MODEL_MAPPING = os.getenv("USE_MODEL_MAPPING", "true").lower() != "false"
 
 AWS_REGION = os.environ.get("AWS_REGION", "us-west-2")
-DEFAULT_MODEL = os.environ.get("DEFAULT_MODEL", "anthropic.claude-3-sonnet-20240229-v1:0")
+DEFAULT_MODEL = os.environ.get("DEFAULT_MODEL", "default")
```

Comment:
We don't need to change this here, since we can do it in our image  (`Makefile`)